### PR TITLE
[script.module.trakt] 3.1.0

### PR DESCRIPTION
**Pull request by Razzeee**
Small update adding two features

### CODE REVIEW

File: script.module.trakt/addon.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.trakt"
        name="trakt.py"
-       version="3.0.0"
+       version="3.1.0"
        provider-name="Fuzeman, Razzeee">
```
Comment (Rechi) [inline script.module.trakt/addon.xml]: correct author is `Dean Gardiner`, https://github.com/fuzeman/trakt.py/blob/3.1.0/setup.cfg

**Comment by Razzeee** [comment]
Updated

File: script.module.trakt/addon.xml [hunk 2]
```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.trakt"
        name="trakt.py"
-       version="3.0.0"
-       provider-name="Fuzeman, Razzeee">
+       version="3.1.0"
+       provider-name="Dean Gardiner, Razzeee">
```
Comment (Rechi) [inline script.module.trakt/addon.xml]: author is only `Dean Gardiner`

File: script.module.trakt/addon.xml [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
-    <import addon="script.module.requests" version="2.12.4"/>
-    <import addon="script.module.six" version="1.9.0"/>
-    <import addon="script.module.arrow" version="0.6.0"/>
+    <import addon="script.module.requests" version="2.15.1"/>
```
Comment (Rechi) [inline script.module.trakt/addon.xml]: Upstream specifies `2.4.0` as minimum version. Why are you now bumping it to an even higher version then before?

File: script.module.trakt/addon.xml [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+    <import addon="script.module.six" version="1.10.0"/>
+    <import addon="script.module.arrow" version="0.10.0"/>
   </requires>
   <extension point="xbmc.python.module"
              library="lib" />
   <extension point="xbmc.addon.metadata">
-    <summary lang="en_GB">Python interface for the trakt.tv API</summary>
-    <description lang="en_GB">Packed for Kodi from https://github.com/fuzeman/trakt.py</description>
+    <summary>Python interface for the trakt.tv API</summary>
+    <description>Packed for Kodi from https://github.com/fuzeman/trakt.py</description>
```
Comment (Rechi) [inline script.module.trakt/addon.xml]: `lang="en_GB"` isn't valid, correct is`lang="en"` for gotham

File: script.module.trakt/addon.xml (hunk 3 above)
Comment (Razzeee) [reply script.module.trakt/addon.xml]: To get the latest version that we have in gotham

Comment (Rechi) [reply script.module.trakt/addon.xml]: So you don't know that version means minimum version and that the repo for a specific Kodi version contains only one version of every add-on.

Comment (Razzeee) [reply script.module.trakt/addon.xml]: Sure, but why not force it to get the latest with all possible bugfixes?

File: script.module.trakt/addon.xml (hunk 2 above)
Comment (Razzeee) [reply script.module.trakt/addon.xml]: Changed

File: script.module.trakt/addon.xml (hunk 3 above)
Comment (Razzeee) [reply script.module.trakt/addon.xml]: Changed

**Pull request merged**